deployments/helm-chart/Chart.yaml

```diff
@@ -11,6 +11,6 @@ keywords:
   - ingress
   - nginx
 maintainers:
-  - name: NGINX Kubernetes Team
+  - name: nginxinc
```

Comment:
Note:
The linting enforces that for every maintainer https://github.com/$MAINTAINER does not 404.

Let me know if there is a better option.